invenio_pidstore/cli.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Click command-line interface for PIDStore management."""
+
+from __future__ import absolute_import, print_function
+
+import click
+from flask_cli import with_appcontext
+from invenio_db import db
+
+
+def process_status(ctx, param, value):
+    """Return status value."""
+    from .models import PIDStatus
+
+    # Allow empty status
+    if value is None:
+        return None
+
+    if value not in PIDStatus.attributes:
+        raise click.BadParameter('Status needs to be one of {0}.'.format(
+            ', '.join(PIDStatus.attributes)
+        ))
+    return getattr(PIDStatus, value)
+
+
+#
+# PIDStore management commands
+#
+
+@click.group()
+def pid():
+    """PIDStore management commands."""
+
+
+@pid.command()
+@click.argument('pid_type')
+@click.argument('pid_value')
+@click.option('-p', '--provider', 'pid_provider', default=None)
```

Comment:
I don't think we should allow specifying the provider. Only PIDs created by a provider should set it.